### DISCUSSION TITLE
Keyboard Shortcuts Support Deprecated from iOS App?

### DISCUSSION THREAD

**Discussion by extratone** [community/community#62298] [Mobile]
**On iPhone**, at least (away from my iPad Pro until EOD, when I will follow up with more details,) the hardware keyboard shortcuts I've relied on (`⌘N` to create new Issues, especially,) are not responding on this build. I have been unable to find any formal documentation of this removal. Apologies if I missed something.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.123.0 (105244228)
Device: iPhone 12 Pro Max (iOS 16.6)
</details>

**Comment by Iron-Ham** [comment]
Hi @extratone,

I haven't been able to replicate that behavior locally -- both with and without full keyboard access enabled on the phone. Could you provide more information on where you're running into this? I was able to get the expected behavior of creating a new issue from the home page -- but in trying to replicate that, I noticed that we don't support that behavior on repository pages with the same shortcut (which I'll correct shortly -- I would expect it in a production release in a few weeks time). 


Thanks,
Hesham

**Reply by extratone** [reply]
I would be glad to but I seem to have lost the reference I made (using the Keyboard Shortcuts guide summoned by holding ⌘ on my iPad Pro) to test against. I'm traveling atm so I can make it again myself in a few days time. Unless you guys happen to have a reference document handy?

I haven't been able to find documentation on the subject in the GitHub docs.

**Reply by akisano** [reply]
It seems iPadOS is recognized as windows/Linux
At least on comment section

https://docs.github.com/en/get-started/using-github/keyboard-shortcuts

It makes emacs keybind unusable...

**Reply by Iron-Ham** [reply]
@akisano that sounds like an issue on mobile web? Could you open up a new discussion around that under the General topic?
